changelogs/fragments/358-vmware_guest_serial_port.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - vmware_guest_serial_port - fixed to have idepomtency(https://github.com/ansible-collections/vmware/pull/358).
```

Comment:
In order to be sure everybody understand, I would avoid the world `idempotency` ( not idepomtency :-) ) and just say: ensure we can run the module two times in a row without unexpected side effect.

Reply:
Thank you for your advice.  
I modified the changelog file that way!